gammapy/scripts/download.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Command line tool to download datasets and notebooks from gammapy-extra GitHub repo."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import click
+
+import sys
+import requests
+import json
+from ..extern.pathlib import Path
+from ..extern.six.moves.urllib.request import urlretrieve
+
+log = logging.getLogger(__name__)
+
+apigitUrl = 'https://api.github.com/repos/gammapy/gammapy-extra/git/trees/master:'
+rawgitUrl = 'https://raw.githubusercontent.com/gammapy/gammapy-extra/master/'
+localfolder = Path('./gammapy-extra')
+
+
+@click.command(name='notebooks')
+def cli_download_notebooks():
+    """Download notebooks"""
+
+    downloadproc = DownloadProcess('notebooks', ['environment.yml'])
+    downloadproc.go()
+
+
+@click.command(name='datasets')
+def cli_download_datasets():
+    """Download datasets"""
+
+    downloadproc = DownloadProcess('datasets', [])
+    downloadproc.go()
+
+
+def create_folder(folder):
```

Comment:
I think this whole `create_folder` can be deleted, and you just put `folder.mkdir(parents=True, exist_ok=True)` and let it do whatever it does?